assets/javascripts/kitten/karl/buttons/garbage-button.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react'
+import Radium, { StyleRoot } from 'radium'
+import PropTypes from 'prop-types'
+import { GarbageIcon } from 'kitten/components/icons/garbage-icon'
+import { ButtonIcon } from 'kitten/components/buttons/button-icon'
+
+class KarlGarbageButtonComponent extends Component {
+  static propTypes = {
+    size: PropTypes.string,
+    modifier: PropTypes.string,
+    isDisabled: PropTypes.bool,
+  }
+
+  static defaultProps = {
+    size: 'tiny',
+    modifier: 'beryllium',
+    isDisabled: null,
+  }
+
+  render() {
+    const size = this.props.size
```

Comment:
Que penses-tu d'utiliser la syntaxe `const { size, modifier, disabled } = this.props` ?